src/Symfony/Component/HttpKernel/Tests/HttpCache/ResponseCacheStrategyTest.php

```diff
@@ -371,7 +429,7 @@ public static function cacheControlMergingProvider()
         ];
 
         yield 'merge max-age and s-maxage' => [
-            ['public' => true, 'max-age' => '60'],
+            ['public' => true, 'max-age' => null, 's-maxage' => '60'],
```

Comment:
If one response has `max-age` and the other has `s-maxage`, the final response should only have a `s-maxage` header and no `max-age` (since we don't know the "private" value for one of the responses).